lib/guest_installation_and_configuration_base.pm

```diff
@@ -772,7 +772,12 @@ sub config_guest_network_bridge_device {
         if ($self->{guest_netaddr} ne 'host-default') {
             $self->write_guest_network_bridge_device_config($_bridge_network, $_bridge_device, 'static', 'auto', '', 'yes');
             my $_bridge_device_config_file = '/etc/sysconfig/network/ifcfg-' . $_bridge_device;
-            script_retry("wicked ifup $_bridge_device_config_file $_bridge_device", retry => 3, die => 0);
+            script_retry("wicked ifup $_bridge_device_config_file $_bridge_device", retry => 3, die => 0) if is_sle;
+            #Not sure if it is a bug, but it works in openSUSE this way.
+            if (is_opensuse) {
+                my $_bridge_ipaddr = script_output ("grep IPADDR $_bridge_device_config_file | cut -d \"'\" -f2");
+                script_retry("brctl addbr $_bridge_device; ip addr add $_bridge_ipaddr dev $_bridge_device; ip link set $_bridge_device up", retry => 3, die => 0);
```

Comment:
Does it mean that tumbleweed guest's network is tested in a way similar to our virt-install.sh script -- setup br123, then set up nat, configure the guest to br123 to get internal /non-public ip? If this is the case, I would highly suggest to change it. We do it that way for sle by historical reasons. And actually the most popular customer user story about this, is to use public bridge , sth like br0, to get public ip for guest. I remember this concurrent guest installation openqa baseclass should support it. 
Julie, would you please switch to that?

Reply:
Why do you do all that manual stuff? The network should be set automatically, not worked around manually. That significantly limits the scope and hides product bugs.

Reply:
talked offline and Alice create a progress ticket: https://progress.opensuse.org/issues/95920